package.json

```diff
@@ -21,7 +21,14 @@
     "test:ui": "vitest --ui",
     "parse": "tsx src/lib/parser/cli.ts",
     "generate-mappings": "node scripts/generate-vanilla-mappings.js",
-    "generate-icons": "tsx scripts/generate-icons.ts && tsx scripts/generate-og-image.ts"
+    "generate-icons": "tsx scripts/generate-icons.ts && tsx scripts/generate-og-image.ts",
+    "mgba:docker:build": "node docker/docker-mgba.js build",
+    "mgba:docker:start": "node docker/docker-mgba.js start",
+    "mgba:docker:stop": "node docker/docker-mgba.js stop",
+    "mgba:docker:test": "node docker/docker-mgba.js test",
+    "mgba:docker:logs": "node docker/docker-mgba.js logs",
+    "mgba:docker:status": "node docker/docker-mgba.js status",
+    "mgba:docker:clean": "node docker/docker-mgba.js clean"
   },
```

Comment:
lets simplify this. this is too many available commands. all i want to be able to do is start and maybe stop the mgba enviroment. lets abstract or remove everthing else away